comfy_extras/nodes_post_processing.py

```diff
@@ -242,7 +243,7 @@ def sharpen(self, image: torch.Tensor, sharpen_radius: int, sigma:float, alpha:
 
         result = torch.clamp(sharpened, 0, 1)
 
-        return (result.to(comfy.model_management.intermediate_device()),)
+        return (result,) if comfy.model_management.mps_mode() else (result.to(comfy.model_management.intermediate_device()),)
```

Comment:
looking at this line makes my eyes bleed, what was wrong with a simple blocking statement for readability?

```
if comfy.model_management.mps_mode():
    return (result,)

return (result.to(comfy.model_management.intermediate_device()),)
```

Other than that the code looks like it should work as intended.

Reply:
changed the code style as you asked